resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -189,6 +189,15 @@ package object config extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("1s")
 
+  private[spark] val CLIENT_REPORT_INCLUDE_DRIVER_LOGS_LINK =
+    ConfigBuilder("spark.yarn.report.includeDriverLogsLink")
```

Comment:
we are trying to not add a bunch of unneeded components to configs. how about spark.yarn.includeDriverLogsLinkInReport